react/Header/UserAccountMenu/UserAccountMenu.js

```diff
@@ -54,7 +54,7 @@ const clearLocalStorage = () => {
   }
 };
 
-const BadgeComponent = () => <Hidden desktop component={NewBadge} key="new" className={styles.newBadge} />;
+const BadgeComponent = () => <Hidden desktop className={styles.newBadge}><Badge strong tone="info" key="new">New</Badge></Hidden>;
```

Comment:
The `key` needs to be hoisted to the `Hidden` component as that is the component requiring an identifier